tests/test_json_schema.py

```diff
@@ -935,11 +935,41 @@ class Model(BaseModel):
 
 def test_decimal_constraints():
```

Comment:
This test no longer test anything related to JSON Schema. I believe we already have plenty of such tests in `test_types.py`